app/controllers/api/v1/meetings_controller.rb

```diff
@@ -97,10 +81,6 @@ def find_room
         @room = Room.find_by!(friendly_id: params[:friendly_id])
```

Comment:
This is irrelevant to this PR, but I think we should revisit our way of fetching resources taking into consideration the provider,